.babelrc

```diff
@@ -1,8 +1,9 @@
 {
-  "presets": ["env"],
+  "presets": [
+    "@babel/preset-env"
+  ],
   "plugins": [
-    "transform-object-assign",
```

Comment:
`Object.assign` transformation is required as Nashorn doesn't support it.

Reply:
Added @babel/plugin-transform-object-assign